src/vm/eventpipeconfiguration.cpp

```diff
@@ -46,11 +46,10 @@ EventPipeConfiguration::~EventPipeConfiguration()
         EX_CATCH { }
         EX_END_CATCH(SwallowAllExceptions);
     }
-
-    if(m_pEnabledProviderList != NULL)
+    if(m_pSession != NULL)
     {
-        delete(m_pEnabledProviderList);
-        m_pEnabledProviderList = NULL;
+        DeleteSession(m_pSession);
```

Comment:
This has me a little confused at who owns the session object. Is it the EventPipeConfiguration's responsibility to delete the session or EventPipe?